collector/collector.go

```
@@ -304,7 +304,7 @@ func readLeaseFile(path string) ([]lease, error) {
 	for scanner.Scan() {
 		activeLease, err := parseLease(scanner.Text())
 		if err != nil {
-			return nil, err
+			continue
```

This change just masks all errors. I would rather detect the duid line and skip it, but keep error reporting as-is otherwise.